changelog_generator.php

```diff
@@ -142,22 +143,46 @@
         $issue['html_url'],
         $issue['user']['login']
     );
+
+    if ($config['group-by-labels']) {
+        $labelNames = array_column($issue['labels'], 'name');
+        sort($labelNames);
+        $labelNames = implode(',', $labelNames);
+
+        $usedLabels[$labelNames]         = isset($usedLabels[$labelNames]) ? $usedLabels[$labelNames] : [];
```

Comment:
Since we don't have any requirement regarding PHP version I didn't use PHP 7 features...